### ISSUE TITLE
Bylaws

### ISSUE THREAD

**Issue by burrrata** [pythonpete32/aragoncoop_governance#1]
Referencing this document
- https://github.com/pythonpete32/aragoncoop_governance/blob/master/bylaws.md

## ARTICLE I.  NAME AND PURPOSE

This directly addresses temp Issue #8: Cooperative DAO Bylaws
- https://github.com/aragoncoop/temp/issues/8

### Name
- I think it's actually the "Aragon Cooperative" ? 
- @kevbot can you confirm the current official name of the group?

### Purpose
- as I mentioned in temp Issue #3, I think it makes sense for the purpose to be to fulfill AGP-40. Then once we do that we can broaden the scope to something more generic and community oriented. 
- https://github.com/aragoncoop/temp/issues/3

### Non-Discrimination Policy
- the idea is solid
- we don't have "staff" or "employment", so maybe edit out that part
- maybe also change "rates of pay" or "other benefits" to "compensation" (because the current workflow only accounts for compensation requests)

## Article II.  TBD?

Why does it skip right to Article III? Where's Article II?

## ARTICLE III.  MEMBERSHIP

This directly addresses temp Issue #2: Membership on-boarding and off-boarding. As such, I think it makes sense for this to be a separate document. For example, there could be a `bylaws` folder in the temp workflow repo, and within that folder there could be `bylaws.md` and `membership-onboarding.md`, `membership-ongoing.md`, and `membership-offboarding.md`.
- https://github.com/aragoncoop/temp/issues/2

Overall this section is great, but I think that a lot of the language/terminology could be confusing for people, esp non-native english speakers. Maybe there should be a 1 sentence summary communicating the intent of each section, and then a more technical breakdown of how exactly that looks?

### Section 1    Powers and Duties of the Members
- Generally looks good. It states that members control and manage the Aragon Cooperative.
- Not sure about all the language, esp stuff like gift solicitation. Is that referring to grant funding or something?

### Section 2    Number of Members and Compensation
- agree to decouple this from bylaws 

### Section 3 
- looks good

### Section 4
- maybe just have this refer to the DAO so that there's 1 source of truth for this?

### Section 5: Meetings
- update Director to member
- sane default for "official" meetings = 1 week. Ad hoc meetings can of course happen outside of this 1 week notification window, but for official important stuff a 1 week update seems reasonable as most people are busy
- also, I'd say that meetings should ONLY be used for discussion, and if so, there always needs to be an async way for members to participate via written mediums so that everyone can contribute. This implies that all decisions are made via voting on the DAO, not via meetings.

### Section 7    Electronic Meetings
- not sure about the conference telephone bit. Maybe just say that prior to any meeting the group needs to agree on the tool/medium used for that meeting so that everyone can participate. What would be a sane default for this?

### Section 8    Resignations, Termination and Absences
- not so sure on a Membership working group, only because I don't think at this moment I would trust anyone to manage that group lol
- I think this would be best in a separate off-boarding document rather than embedded in the bylaws.
- I agree that resignation needs to be presented in writing and there should be a process and place for that.

### Section 9   Onboarding
- Agree that this should be in it's own document

## ARTICLE IV.  WORKING GROUPS
- initially I think we should keep things simple. A flat org. No one and no working groups have additional power. Maybe that's something we could explore in the future, but after we deliver on AGP-40 and have basic bylaws and membership stuff figured out
- yeah I also don't like the idea of a membership working group. Let's keep it flat for now
- governance working group = everyone lol

### Section 1    Documents, Obligations and Disbursements
- agree to separate this from the bylaws - right now this is the temp workflow

## ARTICLE VII. PARLIAMENTARY AUTHORITY

I'm confused as to why this section is called "parliamentary authority?" What authority, and to whom, and why?

Also, I don't think we have or need a fiscal year. If we did have a fiscal year, what problem would that solve? 

Also, everything we do is already transparent on GitHub or the DAO. We don't really need to release transparency reports because everything is out in the open. That being said, it would be good to have a monthly or quarterly newsletter / blog post / update where we communicate what we're doing to the broader Aragon community. Not sure if that fits into this section, but it might be good to encode that somewhere here in the operating manual? 

### Section 1     General
- I think it's best to keep any references to "legal fees" out of this. I could forsee someone using that as a loophole to drain the org very very quickly. Also, we don't have the time or money to handle that. The rule of thumb I stick to is that you should only use the word legal if you are drafting something with a lawyer lol

### Section 2     Limitation of Liability
- we don't have "employees". Everyone is a volunteer. 
- honestly, there's a lot of words there and I'm finding it all hard to follow. Furthermore, we don't have the ability (time, money, jurisdiction) to enforce any of that, esp on a global scale. I think it's best to leave this out for now
- I agree that once the Aragon Court is up and running, we can draft something that could be enforceable within that context 

## ARTICLE IX.  AMENDMENTS

The temp workflow already has a process in place to create Meta Proposals to change the governance process at any time. Also, the Cooperative is meant to be nimble and move fast (relative to ANVs). As such, a yearly review seems unnecessary. I would just omit this part.

**Comment by burrrata** [comment]
I would also add that any decision needs to be made with the Cooperative DAO. This includes, but is not limited to:
- changes to the structure of the DAO (adding/removing apps, adding/removing members, etc)
- changes to the bylaws
- changes to the membership structure
- changes to the workflow

**Comment by burrrata** [comment]
@pythonpete32 I think we're good to close this unless you have further feedback?

**Comment by pythonpete32** [comment]
Nope i think that looks ok
